app/views/transaktions/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>New transaktion</h1>
+
+<%= render "form", transaktion: @transaktion %>
+
+<br>
+
+<div>
+  <%= link_to "Back to transaktions", category_transaktions_path %>
+</div>
```

Comment:
- [ ] Kindly make sure that after a transaction is created users are redirected to the right page, currently, I see an error message after creating  a new transaction see video below ⬇️ 

https://user-images.githubusercontent.com/69638013/198856285-af1ed68b-a99f-4faa-bfb6-86897ca6cf9c.mp4


- Fixing this route will create a better user experience for users 👍🏿

Reply:
> * [ ]  Kindly make sure that after a transaction is created users are redirected to the transactions page for that category, currently, I see an error message after clicking (or taps) the "create transaction" button
> * Fixing this route will create a better user experience for users,   see video below arrow_down
>   https://user-images.githubusercontent.com/69638013/198856285-af1ed68b-a99f-4faa-bfb6-86897ca6cf9c.mp4

- [ ] Kindly as requested by the previous reviewer, make sure  that  a transaction is created without any error and after a transaction is created users are redirected to the transactions page  👇🏻 

![Screenshot from 2022-10-30 03-33-09](https://user-images.githubusercontent.com/79058364/198858626-7bd427b6-5404-4268-a163-298834437fa8.png)